plugins/star-rating/frontend/public/javascripts/countly.views.js

```diff
@@ -502,18 +562,370 @@ window.starView = countlyView.extend({
+                    sType: "string",
+                    "sTitle": "RATING",
+                    "mRender": function (d) {
+                        var ratings = ["<span class='in-table-smiley-wrapper'><img src='/star-rating/images/star-rating/1_color.svg' class='table-detail-rating-img'></span><span class='in-table-smiley-text-wrapper'>Very dissatisfied</span>","<span class='in-table-smiley-wrapper'><img src='/star-rating/images/star-rating/2_color.svg' class='table-detail-rating-img'></span><span class='in-table-smiley-text-wrapper'>Somewhat dissatisfied</span>","<span class='in-table-smiley-wrapper'><img src='/star-rating/images/star-rating/3_color.svg' class='table-detail-rating-img'></span><span class='in-table-smiley-text-wrapper'>Neither satisfied nor dissatisfied</span>","<span class='in-table-smiley-wrapper'><img src='/star-rating/images/star-rating/3_color.svg' class='table-detail-rating-img'></span><span class='in-table-smiley-text-wrapper'>Somewhat satisfied</span>","<span class='in-table-smiley-wrapper'><img src='/images/star-rating/4_color.svg' class='table-detail-rating-img'></span><span class='in-table-smiley-text-wrapper'>Very satisfied</span>"];
+                        return ratings[d-1];
+                    }
+                },
+                {
+                    "mData": function (row) {
+                        if (row.comment) return row.comment;
+                        else return "-";
+                    },
+                    sType: "string",
+                    "sTitle": "COMMENT"
+                },
+                {
+                    "mData": function (row) {
+                        if (row.email) return row.email;
+                        else return "-";
+                    },
+                    sType: "string",
+                    "sTitle": "E-MAIL"},
+                {
+                    "mData": "ts",
+                    sType: "date",
+                    "sTitle": "TIME",
+                    "mRender": function(d) {return moment(d).format('ddd, DD MMM YYYY HH:MM:SS');}
+                }
+            ];
+
+            $('#tableThree').dataTable($.extend({}, $.fn.dataTable.defaults, {
+                "aaData": this.templateData['commentsData'],
+                "aoColumns": columnsDefine
+            }));
+        }
+    },
+    renderFeedbacksTable: function(isRefresh) {
+        var self = this;
+        this.templateData["widgetsData"] = this.getFeedbackWidgetsData();
+        if (isRefresh) {
+            starRatingPlugin.requestFeedbackWidgetsData()
+            .then(function(json) {
+                CountlyHelpers.refreshTable(self.widgetTable, json);    
+            })
+        } else {
+            var columnsDefine = [
+                {
+                    "mData": function (row) {
+                        return row.popup_header_text;   
+                    } ,
+                    sType: "string",
+                    "sTitle": "NAME"
```

Comment:
columns should be localized, try reusing what we have in core dashboard.properties and if not there, then only add new localization strings